src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+            System.out.println(i + ". "  + "[" +  statusIcon + "]" + " " + description);
+            i+= 1;
+        }
+    }
+
+
+    //Mark as done
+    public Boolean markDone(String line, ArrayList<Task> taskList) {
+        String[] words = line.split(" ");
+        int LENG_LIST = words.length;
+        if (LENG_LIST == 1){
+            if (words[0].toUpperCase().equals("DONE")) {
+                System.out.println("Please specify (eg: done 2) or just add a new one");
+                return true;
+            }
+        }
+        Boolean isSet= false;
+        if (words[0].toUpperCase().equals("DONE")) {
+            int IDX_WORDS = Integer.parseInt(words[1]) - 1;
+            if (words[1] == null){
+                return isSet;
+            } else if (IDX_WORDS < 0 || IDX_WORDS > taskList.size() -1) {
+                System.out.println("Index is out of bounds, please try again!");
+                return true;
+            }
+            taskList.get(Integer.parseInt(words[1]) - 1).markIt();
+            String statusIcon = taskList.get(Integer.parseInt(words[1]) - 1).getStatusIcon();
+            String description = taskList.get(Integer.parseInt(words[1]) - 1).getDescription();
+            System.out.println((Integer.parseInt(words[1])) + ". "+ "[" + statusIcon + "]" + " " + description);
+            System.out.println("Done! We have checked " + words[1] + "!");
+            isSet = true;
+            }
+        return isSet;
+    }
+
+
+    //Add a task
+    public void addTask(String line, ArrayList<Task> taskList) {
+
+        String feedIn = line.toUpperCase();
+        String ogString = line;
+        while (!feedIn.equals("BYE")) {
+            if (markDone(ogString, taskList)){
+                System.out.println("___________________________________________________________________________\n");
+            }
+            else if (feedIn.equals("LIST")) {
+                System.out.println("___________________________________________________________________________");
+                this.printTaskList(taskList);
+                System.out.println("___________________________________________________________________________");
+            }
+
+            else {
+                System.out.println("___________________________________________________________________________");
+                Task t = new Task(ogString);
+                taskList.add(t);
+                System.out.println("added: " + ogString + "\n");
+                System.out.println("___________________________________________________________________________");
+            }
+            Scanner scanner = new Scanner(System.in);
+            ogString = scanner.nextLine();
+            feedIn= ogString.toUpperCase();
+
+
+        }
+        System.out.println("Bye. Hope to see you again soon!\n");
+        System.out.println("___________________________________________________________________________");
+
+    }
+
+
+    //Get things started
```

Comment:
it'd better change the comments from `//` to `/**`

Reply:
Yes, agree!